BetterPasswordPlugin.inc.php

```diff
@@ -229,21 +230,17 @@ function handleTemplateDisplay($hookName, $args) {
 		if ($template === 'frontend/pages/userLogin.tpl') {
 			if ($templateMgr->getTemplateVars('error') === 'user.login.loginError') {
 				$username = $templateMgr->getTemplateVars('username');
-				$userDao = DAORegistry::getDAO('UserDAO');
-				$user = $userDao->getByUsername($username);
-				if (isset($user)) {
-					$count = $user->getData($this->getName()."::badPasswordCount");
-					$time = $user->getData($this->getName()."::badPasswordTime");
+				$badpwFailedLoginsDao = DAORegistry::getDAO('BadpwFailedLoginsDAO');
+				$user = $badpwFailedLoginsDao->getByUsername($username);
+				if (!is_null($user)) {
```

Comment:
The change from `isset()` to `!is_null()` here doesn't add any functionality, and clutters the commit.